util/benchmark_utils.ts

```diff
@@ -283,12 +282,15 @@ function extractProxyFields(data: BenchmarkRun[]): void {
       if (!d) continue;
       const name = field + "_proxy";
       const newField = {};
+      // deno-lint-ignore no-explicit-any
```

Comment:
aside: this lint rule is super annoying

Reply:
I do agree, but i think its a good annoyance: `any` is a horrible thing